Resources/public/js/models/ez-locationmodel.js

```diff
@@ -345,6 +345,69 @@ YUI.add('ez-locationmodel', function (Y) {
             return attrs;
         },
 
+        /**
+         * Returns the REST API sort order based on the sortOrder attribute
+         *
+         * @method _getSortDirection
+         * @protected
+         * @return 'ascending' or 'descending'
+         */
+        _getSortDirection: function () {
+            return this.get('sortOrder') === 'ASC' ? 'ascending' : 'descending';
+        },
+
+        /**
+         * Returns the REST API sort clause identifier based on the sortField
+         * attribute.
+         *
+         * @method _getSortClauseIdentifier
+         * @protected
+         * @return {String|Null}
+         */
+        _getSortClauseIdentifier: function () {
+            var sortField = this.get('sortField');
+
+            switch(sortField) {
+                case 'MODIFIED':
+                    return 'DateModified';
+                case 'PUBLISHED':
+                    return 'DatePublished';
+                case 'PATH':
+                    return 'LocationPath';
+                case 'SECTION':
+                    return 'SectionIdentifier';
+                case 'DEPTH':
+                    return 'LocationDepth';
+                case 'CLASS_IDENTIFIER':
+                case 'CLASS_NAME':
+                    console.warn(sortField + ' sort order is unsupported at the moment');
```

Comment:
see: https://github.com/ezsystems/PlatformUIBundle/pull/622#discussion_r68007200

If you return null or not I'm not sure of, I assume null means no sorting, but could behave like default.